test/functional/plugin/lsp/fake_lsp_server.lua

```diff
@@ -0,0 +1,121 @@
+-- luacheck: push ignore 113
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
just putting this here to see whether test would pass (warning about buster global). What is the proper way to share this startup code between spec files?

Reply:
Why can't we put this file in fixtures like it was before?

Reply:
I feel like these files should be swapped, the core helper code should be in fixtures and the actual tests should be here. @mfussenegger wdyt?

Reply:
the reason why I didn't put it into the fixtures was that all fixtures seemed to be Lua code for Neovim-in-process. I didn't see example code for test setup except from `helpers.lua` in the root folder

Reply:
Right but you still have [test/functional/fixtures/fake-lsp-server.lua](https://github.com/neovim/neovim/pull/15723#diff-3319ec2c423f139a0da97179848b61fc4a17dc77951ccfe22697699992285106) in fixtures,

Reply:
I don't fully understand how this should be resolved. We have two files:
- `test/functional/plugin/lsp/fake_lsp_server.lua` which should maybe be renamed in `fake_lsp_server_launch`. It was extracted from the tests. It needs to be executed by busted and required in code executed by busted.
- `test/functional/fixtures/fake-lsp-server.lua` which is executed by Neovim in-process

Where should I move which of the files?